packages/smooth_app/lib/pages/product/simple_input_text_field.dart

```diff
@@ -65,7 +67,7 @@ class SimpleInputTextField extends StatelessWidget {
                   return <String>[];
                 }
 
-                return manager.getSuggestions(input);
+                return manager!.getSuggestions(input);
```

Comment:
Returning a nullable object in the build method which can and is in some cases is null doesn't seem like best praxis 🤔

Reply:
@M123-dev You are :)
The manager is null only if tag type is null. And we `manager!` only if tag type is _not_ null (check above, first test).